core/modules/taxonomy/taxonomy_vocabulary.class.inc

```diff
@@ -55,6 +55,11 @@ class TaxonomyVocabulary {
    */
   public $weight = 0;
 
+  /**
+   * @var null|TaxonomyVocabulary
```

Comment:
Is there a reason why `null` is first? It might read better to put it like: `@var TaxonomyVocabulary|null` if we consider `null` as the fallback.

Reply:
No reason. Sure, let's switch them.